ch_pipeline/processing/quarterstack.py

```diff
@@ -219,14 +198,34 @@
     # Save the ringmap out to a ZarrZip file
     - type: draco.core.io.SaveZarrZip
       in: ringmap_trunc
-      out: zip_handle
+      out: ringmap_zip_handle
       params:
         output_name: "ringmap.zarr.zip"
 
+    # Mask out the bright sources so we can see the high delay structure more easily
+    - type: ch_pipeline.analysis.flagging.MaskSource
+      in: sstack_fix
+      out: sstack_flag_src
+      params:
+        source: ["CAS_A", "CYG_A", "TAU_A", "VIR_A"]
+
+    # Try and derive an optimal time-freq factorizable mask that covers the
+    # existing masked entries
+    - type: draco.analysis.flagging.MaskFreq
+      in: sstack_flag_src
+      out: factmask
+      params:
+        factorize: true
```

Comment:
don't we want to mention the known freq band which are bad. 

bad_freq_ind: [[738, 753], [703, 720], [490, 509], [857, 873], [485, 488]]

Reply:
All of these bands should now be in the static mask, so I think as long as we aren't doing any blending then we don't need to mask them. 

I'm also thinking that for the stacks it's better not to do any additional masking that isn't in the actual quarterstack files, so that we can easily see if there's a bad a band that made it through before stacking